gen/calc.py

```diff
@@ -622,13 +622,11 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
             'rexray': {
                 'loglevel': 'info',
                 'modules': {
-                    'default-admin': {
```

Comment:
Because RR doesn't use a port anymore, it use a socket. Or maybe I'm making a mistake 🤔